src/Storage/Services/RegisterService.cs

```diff
@@ -111,11 +111,15 @@ public async Task<int> PartyLookup(string person, string orgNo)
 
                 return -1;
             }
-            else
+            else if (response.StatusCode == HttpStatusCode.BadRequest)
```

Comment:
not a buge fan of this exception throwing. Now that validation of input has been added to the controller we should not be getting bad requests. And if we do, it can be handled like any platform exception.

Reply:
My initial thought were that any BadRequest response we get from Register should result in a BadRequest response from Storage, but that might not be the case. I'm now inclined to agree with @acn-sbuad

Reply:
Yep! I was thinking about the same as well, which actually makes this test obsolete! 👍 

https://github.com/Altinn/altinn-storage/pull/403#discussion_r1601382755